mllm/models/qwen3_5/configuration_qwen3_5.hpp

```
@@ -0,0 +1,128 @@
+  int64_t im_start_token_id = 248045;
+  int64_t im_end_token_id = 248046;
+  int64_t thinking_start_token_id = 248068;
+  int64_t thinking_end_token_id = 248069;
+
+  bool tie_word_embeddings = true;
+  int32_t max_cache_length = 2048;
+
+  aops::LinearImplTypes linear_impl_type = aops::LinearImplTypes::kDefault;
+
+  // Helpers
+  bool isFullAttentionLayer(int layer_idx) const {
```

Clang (14.0.6)

[error] function 'isFullAttentionLayer' should be marked [[nodiscard]]

(modernize-use-nodiscard,-warnings-as-errors)